meson.build

```diff
@@ -23,6 +23,7 @@ wayland_client = dependency('wayland-client')
 wayland_server = dependency('wayland-server')
 wayland_protos = dependency('wayland-protocols', version: '>=1.14')
 math = cc.find_library('m', required: false)
+gbm = dependency('gbm', required: false)
```

Comment:
Can avoid the `if` by adding `disabler: true` here.

Reply:
Done, required meson >= 0.49.